<!DOCTYPE html><html><head><meta name='viewport' content='width=device-width, initial-scale=1'><meta name="description" content="A bioinformatic tool for identifying novel redox-active disulphide bonds in mammalian proteins, utilizing RCSB protein databank structures and providing a redox score."><meta name="keywords" content="ReDisulphID, bioinformatics, protein function, redox-active disulphides, covalent drugs, mammalian proteins, RCSB protein databank, redox score, HSE, solvent accessibility, pKa, disulphide bond prediction"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="robots" content="index, follow"><link rel='icon'  type='image/png' href='icon.png'><link rel = 'stylesheet' href = 'style.css'><script src="jquery-3.7.1.min.js"></script><script src='3Dmol-min.js'></script><title>ReDisulphID</title></head><body><a href = 'list.html'><div class= 'title'>ReDisulphID<div class='subtitle'><p>a tool for identifying redox-active disulphides from structures</div></div></a><div class = 'main'>
<div class = 'menu'><a class = 'button' href = 'index.html'><div class= 'menuitem'>Home</div></a><a class = 'button' href = 'about.html'><div class= 'menuitem'>About</div></a><a class = 'button' href = 'list.html'><div class= 'menuitem'>List</div></a></div><h1>Deformed epidermal autoregulatory factor 1 homolog</h1><div class = 'tabs'><div class = 'sep'>Intramolecular</div><a href = '#0' class = 'tablink'><div class = 'item' style='background:rgba(142.41036524070145,255,100,0.4);'>Cysteine 504 and cysteine 528</div></a><a href = '#1' class = 'tablink'><div class = 'item' style='background:rgba(144.42682374263705,255,100,0.4);'>Cysteine 504 and cysteine 524</div></a><a href = '#2' class = 'tablink'><div class = 'item' style='background:rgba(144.70548178413347,255,100,0.4);'>Cysteine 504 and cysteine 507</div></a><a href = '#3' class = 'tablink'><div class = 'item' style='background:rgba(148.68804664723032,255,100,0.4);'>Cysteine 515 and cysteine 540 <span class = 'lig_tab' title = 'Ligandable'>L</span></div></a><a href = '#4' class = 'tablink'><div class = 'item' style='background:rgba(150.40698360268962,255,100,0.4);'>Cysteine 518 and cysteine 540</div></a><a href = '#5' class = 'tablink'><div class = 'item' style='background:rgba(153.17155213839501,255,100,0.4);'>Cysteine 515 and cysteine 518 <span class = 'lig_tab' title = 'Ligandable'>L</span></div></a><a href = '#6' class = 'tablink'><div class = 'item' style='background:rgba(155.0529004013134,255,100,0.4);'>Cysteine 524 and cysteine 528</div></a><a href = '#7' class = 'tablink'><div class = 'item' style='background:rgba(165.8428720083247,255,100,0.4);'>Cysteine 507 and cysteine 528</div></a><a href = '#8' class = 'tablink'><div class = 'item' style='background:rgba(167.0378619153675,255,100,0.4);'>Cysteine 507 and cysteine 524</div></a><a href = '#9' class = 'tablink'><div class = 'item' style='background:rgba(nan,nan,100,0.4);'>Cysteine 541 and cysteine 112</div></a><a  href = 'javascript:void(0);' onclick = 'remove(this)' class = 'extra_tabs'>More...</a><a href = '#10' class = 'tablink'><div class = 'item' style='background:rgba(nan,nan,100,0.4);'>Cysteine 525 and cysteine 137</div></a><a href = '#11' class = 'tablink'><div class = 'item' style='background:rgba(nan,nan,100,0.4);'>Cysteine 228 and cysteine 541</div></a><a href = '#12' class = 'tablink'><div class = 'item' style='background:rgba(nan,nan,100,0.4);'>Cysteine 228 and cysteine 112</div></a><a href = '#13' class = 'tablink'><div class = 'item' style='background:rgba(nan,nan,100,0.4);'>Cysteine 484 and cysteine 137</div></a></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>2jw6 A 504 A 528 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(142.41036524070145,255,100,0.4);'>A redox-regulated disulphide may form within <b>Deformed epidermal autoregulatory factor 1 homolog</b> between cysteines 504 and 528. </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 60, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>90</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/2jw6' ><div class = 'item info'>2jw6</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>solution structure of the deaf1 mynd domain</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2007-10-08</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>4</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>49</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>6</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>0</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/O75398/entry'><div class = 'item info'>O75398</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>504</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>528</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Deformed epidermal autoregulatory factor 1 homolog</div></div><h2>Ligandability</h2><h3>Cysteine 504 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div><h3>Cysteine 528 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>2jw6 A 504 A 524 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(144.42682374263705,255,100,0.4);'>A redox-regulated disulphide may form within <b>Deformed epidermal autoregulatory factor 1 homolog</b> between cysteines 504 and 524. </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 60, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>88</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/2jw6' ><div class = 'item info'>2jw6</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>solution structure of the deaf1 mynd domain</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2007-10-08</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>4</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>47</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>6</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>0</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/O75398/entry'><div class = 'item info'>O75398</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>504</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>524</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Deformed epidermal autoregulatory factor 1 homolog</div></div><h2>Ligandability</h2><h3>Cysteine 504 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div><h3>Cysteine 524 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>2jw6 A 504 A 507 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(144.70548178413347,255,100,0.4);'>A redox-regulated disulphide may form within <b>Deformed epidermal autoregulatory factor 1 homolog</b> between cysteines 504 and 507. </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 60, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>88</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/2jw6' ><div class = 'item info'>2jw6</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>solution structure of the deaf1 mynd domain</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2007-10-08</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>4</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>42</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>6</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>0</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/O75398/entry'><div class = 'item info'>O75398</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>504</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>507</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Deformed epidermal autoregulatory factor 1 homolog</div></div><h2>Ligandability</h2><h3>Cysteine 504 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div><h3>Cysteine 507 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>4a24 A 515 A 540 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(148.68804664723032,255,100,0.4);'>A redox-regulated disulphide may form within <b>Deformed epidermal autoregulatory factor 1 homolog</b> between cysteines 515 and 540. </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 60, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>86</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/4a24' ><div class = 'item info'>4a24</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>structural and functional analysis of the deaf-1 and bs69 mynd domains</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2011-09-22</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>4</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>50</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>8</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>1</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/O75398/entry'><div class = 'item info'>O75398</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>515</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>540</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Deformed epidermal autoregulatory factor 1 homolog</div></div><h2>Ligandability</h2><h3>Cysteine 515 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability ligandable'><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div><h3>Cysteine 540 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>2jw6 A 518 A 540 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(150.40698360268962,255,100,0.4);'>A redox-regulated disulphide may form within <b>Deformed epidermal autoregulatory factor 1 homolog</b> between cysteines 518 and 540. </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 60, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>85</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/2jw6' ><div class = 'item info'>2jw6</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>solution structure of the deaf1 mynd domain</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2007-10-08</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>4</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>39</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>8</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>0</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/O75398/entry'><div class = 'item info'>O75398</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>518</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>540</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Deformed epidermal autoregulatory factor 1 homolog</div></div><h2>Ligandability</h2><h3>Cysteine 518 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div><h3>Cysteine 540 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>2jw6 A 515 A 518 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(153.17155213839501,255,100,0.4);'>A redox-regulated disulphide may form within <b>Deformed epidermal autoregulatory factor 1 homolog</b> between cysteines 515 and 518. </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 60, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>83</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/2jw6' ><div class = 'item info'>2jw6</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>solution structure of the deaf1 mynd domain</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2007-10-08</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>4</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>49</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>8</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>2</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/O75398/entry'><div class = 'item info'>O75398</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>515</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>518</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Deformed epidermal autoregulatory factor 1 homolog</div></div><h2>Ligandability</h2><h3>Cysteine 515 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability ligandable'><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div><h3>Cysteine 518 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>2jw6 A 524 A 528 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(155.0529004013134,255,100,0.4);'>A redox-regulated disulphide may form within <b>Deformed epidermal autoregulatory factor 1 homolog</b> between cysteines 524 and 528. </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 60, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>82</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/2jw6' ><div class = 'item info'>2jw6</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>solution structure of the deaf1 mynd domain</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2007-10-08</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>4</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>48</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>7</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>0</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/O75398/entry'><div class = 'item info'>O75398</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>524</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>528</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Deformed epidermal autoregulatory factor 1 homolog</div></div><h2>Ligandability</h2><h3>Cysteine 524 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div><h3>Cysteine 528 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>2jw6 A 507 A 528 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(165.8428720083247,255,100,0.4);'>A redox-regulated disulphide may form within <b>Deformed epidermal autoregulatory factor 1 homolog</b> between cysteines 507 and 528. </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 60, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>77</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/2jw6' ><div class = 'item info'>2jw6</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>solution structure of the deaf1 mynd domain</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2007-10-08</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>4</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>43</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>10</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>0</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/O75398/entry'><div class = 'item info'>O75398</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>507</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>528</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Deformed epidermal autoregulatory factor 1 homolog</div></div><h2>Ligandability</h2><h3>Cysteine 507 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div><h3>Cysteine 528 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>4a24 A 507 A 524 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(167.0378619153675,255,100,0.4);'>A redox-regulated disulphide may form within <b>Deformed epidermal autoregulatory factor 1 homolog</b> between cysteines 507 and 524. </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 60, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>76</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/4a24' ><div class = 'item info'>4a24</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>structural and functional analysis of the deaf-1 and bs69 mynd domains</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2011-09-22</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>4</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>49</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>9</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>0</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/O75398/entry'><div class = 'item info'>O75398</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>507</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>524</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Deformed epidermal autoregulatory factor 1 homolog</div></div><h2>Ligandability</h2><h3>Cysteine 507 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div><h3>Cysteine 524 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>2mbv A 90 A 112 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(nan,nan,100,0.4);'>A redox-regulated disulphide may form within <b>Deformed epidermal autoregulatory factor 1 homolog</b> between cysteines 541 and 112 (90 and 112 respectively in this structure). </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 60, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>nan</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/2mbv' ><div class = 'item info'>2mbv</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>lmo4-lim2 in complex with deaf1 (404-418)</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2013-08-06</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>4</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>50</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>8</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>18</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/Q9Z1T5/entry'><div class = 'item info'>Q9Z1T5</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>541</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>112</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Deformed epidermal autoregulatory factor 1 homolog</div></div><h2>Ligandability</h2><h3>Cysteine 90 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div><h3>Cysteine 112 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div><div class = 'item infotitle'><b>Cysteine 112 in protein B could not be asigned to a Uniprot residue.</b></div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>2mbv A 115 A 137 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(nan,nan,100,0.4);'>A redox-regulated disulphide may form within <b>Deformed epidermal autoregulatory factor 1 homolog</b> between cysteines 525 and 137 (115 and 137 respectively in this structure). </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 60, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>nan</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/2mbv' ><div class = 'item info'>2mbv</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>lmo4-lim2 in complex with deaf1 (404-418)</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2013-08-06</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>4</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>61</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>7</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>10</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/Q9Z1T5/entry'><div class = 'item info'>Q9Z1T5</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>525</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>137</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Deformed epidermal autoregulatory factor 1 homolog</div></div><h2>Ligandability</h2><h3>Cysteine 115 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div><h3>Cysteine 137 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div><div class = 'item infotitle'><b>Cysteine 137 in protein B could not be asigned to a Uniprot residue.</b></div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>2mbv A 87 A 90 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(nan,nan,100,0.4);'>A redox-regulated disulphide may form within <b>Deformed epidermal autoregulatory factor 1 homolog</b> between cysteines 228 and 541 (87 and 90 respectively in this structure). </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 60, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>nan</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/2mbv' ><div class = 'item info'>2mbv</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>lmo4-lim2 in complex with deaf1 (404-418)</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2013-08-06</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>4</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>57</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>7</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>22</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/Q9Z1T5/entry'><div class = 'item info'>Q9Z1T5</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>228</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>541</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Deformed epidermal autoregulatory factor 1 homolog</div></div><h2>Ligandability</h2><h3>Cysteine 87 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div><h3>Cysteine 90 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div><div class = 'item infotitle aux'>Uncertain whether structure cysteine 87 has been assigned to correct residue.</div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>2mbv A 87 A 112 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(nan,nan,100,0.4);'>A redox-regulated disulphide may form within <b>Deformed epidermal autoregulatory factor 1 homolog</b> between cysteines 228 and 112 (87 and 112 respectively in this structure). </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 60, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>nan</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/2mbv' ><div class = 'item info'>2mbv</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>lmo4-lim2 in complex with deaf1 (404-418)</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2013-08-06</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>4</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>65</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>7</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>31</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/Q9Z1T5/entry'><div class = 'item info'>Q9Z1T5</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>228</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>112</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Deformed epidermal autoregulatory factor 1 homolog</div></div><h2>Ligandability</h2><h3>Cysteine 87 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div><h3>Cysteine 112 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div><div class = 'item infotitle aux'>Uncertain whether structure cysteine 87 has been assigned to correct residue.</div><div class = 'item infotitle'><b>Cysteine 112 in protein B could not be asigned to a Uniprot residue.</b></div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>2mbv A 118 A 137 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(nan,nan,100,0.4);'>A redox-regulated disulphide may form within <b>Deformed epidermal autoregulatory factor 1 homolog</b> between cysteines 484 and 137 (118 and 137 respectively in this structure). </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 60, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>nan</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/2mbv' ><div class = 'item info'>2mbv</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>lmo4-lim2 in complex with deaf1 (404-418)</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2013-08-06</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>4</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>48</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>9</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>4</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/Q9Z1T5/entry'><div class = 'item info'>Q9Z1T5</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>484</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>137</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Deformed epidermal autoregulatory factor 1 homolog</div></div><h2>Ligandability</h2><h3>Cysteine 118 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div><h3>Cysteine 137 of Deformed epidermal autoregulatory factor 1 homolog</h3><div class = 'grid'><div class = 'item ligandability '><p><a href = 'https://pubs.acs.org/doi/10.1021/jacs.1c11053' target='_blank' rel='noopener'>Not ligandable in the dataset of Yang <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202000870' target='_blank' rel='noopener'>Not ligandable in the dataset of Yan <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/nature18002' target='_blank' rel='noopener'>Not ligandable in the dataset of Backus <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1016/j.cell.2020.07.001' target='_blank' rel='noopener'>Not ligandable in the dataset of Vinogradova <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://doi.org/10.1021/acs.analchem.0c04726' target='_blank' rel='noopener'>Not ligandable in the dataset of Cao <i>et al.</i></a></p></div><div class = 'item ligandability '><p><a href = 'https://www.nature.com/articles/s41587-020-00778-3' target='_blank' rel='noopener'>Not ligandable in the dataset of Kuljanin <i>et al.</i></a></p></div></div><div class = 'item infotitle'><b>Cysteine 137 in protein B could not be asigned to a Uniprot residue.</b></div></div><div class = 'citation'>If this tool was useful for finding a disulphide, please cite:    <p></div></div><script src='search.js'></script></body></html>