<!DOCTYPE html><html><head><meta name='viewport' content='width=device-width, initial-scale=1'><meta name="description" content="A bioinformatic tool for identifying novel redox-active disulphide bonds in mammalian proteins, utilizing RCSB protein databank structures and providing a redox score."><meta name="keywords" content="ReDisulphID, bioinformatics, protein function, redox-active disulphides, covalent drugs, mammalian proteins, RCSB protein databank, redox score, HSE, solvent accessibility, pKa, disulphide bond prediction"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="robots" content="index, follow"><link rel='icon'  type='image/png' href='icon.png'><link rel = 'stylesheet' href = 'style.css'><script src='3Dmol-min.js'></script><title>ReDisulphID</title></head><body><a href = 'list.html'><div class= 'title'>ReDisulphID<div class='subtitle'><p>a tool for identifying redox-active disulphides from structures</div></div></a><div class = 'main'>
<div class = 'menu'><a class = 'button' href = 'index.html'><div class= 'menuitem'>Home</div></a><a class = 'button' href = 'about.html'><div class= 'menuitem'>About</div></a><a class = 'button' href = 'list.html'><div class= 'menuitem'>List</div></a></div><h1>Nuclear protein localization protein 4 homolog</h1><div class = 'tabs'><div class = 'sep'>Intramolecular</div><a href = '#0' class = 'tablink'><div class = 'item' style='background:rgba(159.93477170095332,255,100,0.4);'>Cysteine 341 and cysteine 399</div></a><a href = '#1' class = 'tablink'><div class = 'item' style='background:rgba(169.68325791855204,255,100,0.4);'>Cysteine 586 and cysteine 600</div></a><a href = '#2' class = 'tablink'><div class = 'item' style='background:rgba(170.86571964620742,255,100,0.4);'>Cysteine 130 and cysteine 141</div></a><a href = '#3' class = 'tablink'><div class = 'item' style='background:rgba(170.8886208283072,255,100,0.4);'>Cysteine 130 and cysteine 138</div></a><a href = '#4' class = 'tablink'><div class = 'item' style='background:rgba(171.0720515228767,255,100,0.4);'>Cysteine 586 and cysteine 589</div></a><a href = '#5' class = 'tablink'><div class = 'item' style='background:rgba(171.80972914701522,255,100,0.4);'>Cysteine 600 and cysteine 603</div></a><a href = '#6' class = 'tablink'><div class = 'item' style='background:rgba(172.60051441721944,255,100,0.4);'>Cysteine 138 and cysteine 141</div></a><a href = '#7' class = 'tablink'><div class = 'item' style='background:rgba(173.3986128110975,255,100,0.4);'>Cysteine 586 and cysteine 603</div></a><a href = '#8' class = 'tablink'><div class = 'item' style='background:rgba(174.44246818990285,255,100,0.4);'>Cysteine 589 and cysteine 600</div></a><a href = '#9' class = 'tablink'><div class = 'item' style='background:rgba(177.00957934194085,255,100,0.4);'>Cysteine 589 and cysteine 603</div></a><a href = '#10' class = 'tablink'><div class = 'item' style='background:rgba(208.77681349271327,255,100,0.4);'>Cysteine 341 and cysteine 391</div></a></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>7wwq A 188 A 399 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(159.93477170095332,255,100,0.4);'>A redox-regulated disulphide may form within <b>Nuclear protein localization protein 4 homolog</b> at these cysteines. </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 70.83, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>79.72</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/7wwq' ><div class = 'item info'>7wwq</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>crystal structure of human ufd1-npl4 complex</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2022-02-14</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>4.32</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>nan</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>9.74</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>16.5</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/Q8TAT6/entry'><div class = 'item info'>Q8TAT6</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>341 (Residue 188 in this structure)</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>399 (Residue 399 in this structure)</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Nuclear protein localization protein 4 homolog</div></div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>1nj3 A 9 A 23 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(169.68325791855204,255,100,0.4);'>A redox-regulated disulphide may form within <b>Nuclear protein localization protein 4 homolog</b> at these cysteines. </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 70.83, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>75.14</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/1nj3' ><div class = 'item info'>1nj3</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>structure and ubiquitin interactions of the conserved nzf domain of npl4</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2002-12-30</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>3.73</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>49</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>nan</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>nan</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/Q9ES54/entry'><div class = 'item info'>Q9ES54</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>586 (Residue 9 in this structure)</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>600 (Residue 23 in this structure)</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Nuclear protein localization protein 4 homolog</div></div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>7wwp A 130 A 141 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(170.86571964620742,255,100,0.4);'>A redox-regulated disulphide may form within <b>Nuclear protein localization protein 4 homolog</b> at these cysteines. </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 70.83, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>74.62</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/7wwp' ><div class = 'item info'>7wwp</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>crystal structure of human npl4</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2022-02-14</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>3.99</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>42</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>nan</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>nan</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/Q8TAT6/entry'><div class = 'item info'>Q8TAT6</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>130 (Residue 130 in this structure)</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>141 (Residue 141 in this structure)</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Nuclear protein localization protein 4 homolog</div></div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>7wwp A 130 A 138 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(170.8886208283072,255,100,0.4);'>A redox-regulated disulphide may form within <b>Nuclear protein localization protein 4 homolog</b> at these cysteines. </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 70.83, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>74.61</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/7wwp' ><div class = 'item info'>7wwp</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>crystal structure of human npl4</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2022-02-14</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>3.75</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>38</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>nan</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>nan</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/Q8TAT6/entry'><div class = 'item info'>Q8TAT6</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>130 (Residue 130 in this structure)</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>138 (Residue 138 in this structure)</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Nuclear protein localization protein 4 homolog</div></div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>1nj3 A 9 A 12 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(171.0720515228767,255,100,0.4);'>A redox-regulated disulphide may form within <b>Nuclear protein localization protein 4 homolog</b> at these cysteines. </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 70.83, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>74.53</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/1nj3' ><div class = 'item info'>1nj3</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>structure and ubiquitin interactions of the conserved nzf domain of npl4</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2002-12-30</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>3.74</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>42</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>nan</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>nan</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/Q9ES54/entry'><div class = 'item info'>Q9ES54</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>586 (Residue 9 in this structure)</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>589 (Residue 12 in this structure)</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Nuclear protein localization protein 4 homolog</div></div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>1nj3 A 23 A 26 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(171.80972914701522,255,100,0.4);'>A redox-regulated disulphide may form within <b>Nuclear protein localization protein 4 homolog</b> at these cysteines. </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 70.83, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>74.21</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/1nj3' ><div class = 'item info'>1nj3</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>structure and ubiquitin interactions of the conserved nzf domain of npl4</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2002-12-30</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>3.75</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>43</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>nan</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>nan</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/Q9ES54/entry'><div class = 'item info'>Q9ES54</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>600 (Residue 23 in this structure)</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>603 (Residue 26 in this structure)</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Nuclear protein localization protein 4 homolog</div></div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>7wwp A 138 A 141 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(172.60051441721944,255,100,0.4);'>A redox-regulated disulphide may form within <b>Nuclear protein localization protein 4 homolog</b> at these cysteines. </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 70.83, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>73.87</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/7wwp' ><div class = 'item info'>7wwp</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>crystal structure of human npl4</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2022-02-14</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>3.89</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>42</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>nan</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>nan</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/Q8TAT6/entry'><div class = 'item info'>Q8TAT6</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>138 (Residue 138 in this structure)</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>141 (Residue 141 in this structure)</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Nuclear protein localization protein 4 homolog</div></div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>1q5w A 9 A 26 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(173.3986128110975,255,100,0.4);'>A redox-regulated disulphide may form within <b>Nuclear protein localization protein 4 homolog</b> at these cysteines. </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 70.83, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>73.53</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/1q5w' ><div class = 'item info'>1q5w</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>ubiquitin recognition by npl4 zinc-fingers</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2003-08-11</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>3.74</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>66</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>nan</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>nan</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/Q9ES54/entry'><div class = 'item info'>Q9ES54</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>586 (Residue 9 in this structure)</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>603 (Residue 26 in this structure)</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Nuclear protein localization protein 4 homolog</div></div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>1q5w A 12 A 23 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(174.44246818990285,255,100,0.4);'>A redox-regulated disulphide may form within <b>Nuclear protein localization protein 4 homolog</b> at these cysteines. </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 70.83, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>73.09</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/1q5w' ><div class = 'item info'>1q5w</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>ubiquitin recognition by npl4 zinc-fingers</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2003-08-11</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>3.94</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>70</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>nan</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>nan</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/Q9ES54/entry'><div class = 'item info'>Q9ES54</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>589 (Residue 12 in this structure)</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>600 (Residue 23 in this structure)</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Nuclear protein localization protein 4 homolog</div></div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>1q5w A 12 A 26 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(177.00957934194085,255,100,0.4);'>A redox-regulated disulphide may form within <b>Nuclear protein localization protein 4 homolog</b> at these cysteines. </div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 70.83, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>72.03</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/1q5w' ><div class = 'item info'>1q5w</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>ubiquitin recognition by npl4 zinc-fingers</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2003-08-11</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>4.16</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>71</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>nan</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>nan</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/Q9ES54/entry'><div class = 'item info'>Q9ES54</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>589 (Residue 12 in this structure)</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>603 (Residue 26 in this structure)</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Nuclear protein localization protein 4 homolog</div></div></div><div class = 'tab'><div class = 'item structure' style='background:rgba(200, 200, 200, 0.4);;'><p class = 'struc' hidden>7wwq A 341 A 391 </p> <div style='height: 100%; position: relative;' class='viewer_3Dmoljs viewer' data-backgroundcolor='0xffffff' data-style='stick' data-ui='false'></div></div><div class = 'item redoxstatement' style='background:rgba(208.77681349271327,255,100,0.4);'>A redox-regulated disulphide may form within <b>Nuclear protein localization protein 4 homolog</b> at these cysteines. However, the redox score of this cysteine pair is lower than any known redox-active intermolecular disulphide. <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 70.83, while the lowest score overall is 0.' class = 'help' href = 'about.html#scoreinfo'>?</a></div><h2>Details</h2><div class = 'grid'><div class = 'item infotitle'>Redox score <a title = 'The redox score is related to how likely a cysteine pair is to form a redox active disulphide. It is calculated from solvent accessibility, pKa, and distance between each thiol in the cysteine pair. The highest redox score is 100.0, and the lowest redox score of a known redox-active intermolecular disulphide is 70.83, while the lowest score overall is 0.' href = 'about.html#scoreinfo'>?</a></div><div class = 'item info'>61.07</div><div class = 'item infotitle'>PDB code</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.rcsb.org/structure/7wwq' ><div class = 'item info'>7wwq</div></a><div class = 'item infotitle'>Structure name</div><div class = 'item info'>crystal structure of human ufd1-npl4 complex</div><div class = 'item infotitle'>Structure deposition date </div><div class = 'item info'>2022-02-14</div><div class = 'item infotitle'>Thiol separation (&#8491;)</div><div class = 'item info'>5.65</div><div class = 'item infotitle'>Half-sphere exposure sum <a title = 'HSE is a measure of the accessibility of an amino acid side-chain. C&#x3B1; atoms are counted in a 12 &#8491; radius around the C&#x3B1; of each cysteine in the potential redox-active disulphide bond. A lower HSE means an amino acid has fewer proximal amino acids, so is more accessible.' href = 'about.html#hseinfo'>?</a></div><div class = 'item info'>98</div><div class = 'item infotitle'>Minimum pK<sub>a</sub> <a title = 'The lowest pKa from each thiol in the cystiene pair. A lower minimum pKa means a cysteine pair is more likely to form a redox-regulated disulphide bond. The pKa of the cysteine thiols in the potential redox-active disulphide bond are calculated using the PROPKA algorithm.' href = 'about.html#pkainfo'>?</a></div><div class = 'item info'>12.54</div><div class = 'item infotitle'>% buried</div><div class = 'item info'>100</div><div class = 'item infotitle'>Peptide accession</div><a class = 'link' target='_blank' rel='noopener' href = 'https://www.uniprot.org/uniprotkb/Q8TAT6/entry'><div class = 'item info'>Q8TAT6</div></a><div class = 'item infotitle'>Residue number A</div><div class = 'item info'>341 (Residue 341 in this structure)</div><div class = 'item infotitle'>Residue number B</div><div class = 'item info'>391 (Residue 391 in this structure)</div><div class = 'item infotitle'>Peptide name</div><div class = 'item info'>Nuclear protein localization protein 4 homolog</div></div></div><div class = 'citation'>If this tool was useful for finding a disulphide, please cite:    <p></div></div><script src='search.js'></script></body></html>